content/blog/announcing-crossguard-preview/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Announcing Crossguard Preview"
+date: 2019-12-02
+meta_desc: "Today we are announcing Pulumi CrossGuard, a Policy as Code solution, is available all Team Pro and Enterprise customers to preview."
+meta_image: meta.png
+authors: ["erin-krengel"]
+tags: ["crossguard", "policy-as-code", "New-Features", "Pulumi-News","Features"]
+---
+
+Over the past few months, we have been hard at work on Pulumi CrossGuard, a Policy as Code solution. Using the `pulumi-policy` library, you can express flexible business and security rules using code. CrossGuard enables organization administrators to enforce these policies across their organization or just on specific stacks.
+
+<!--more-->
+
+You can run policies against any Pulumi stack, independent of the stack's language. This means you can have policies written in TypeScript and run them against a Python stack. Policies are grouped in Policy Packs and run against a stack during both `pulumi preview` and `pulumi up`. Each resource in a stack is run through the Policy Pack prior to creation, meaning you can block the creation of out-of-compliance resources.
```

Comment:
Nit: "Python **-based** stack". 

Since you are introducing a new term, consider putting "Policy Packs" in italics the first time it is used.